CodingTracker/CodingTracker/Application.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace CodingTracker;
+
+public class Application
+{
+    public void Run(string connectionString)
+    {
+        bool continueRunning = true;
+        CrudMenu crudMenu = new CrudMenu();
+
+        while (continueRunning)
+        {
+            CrudMenuOptions crudMenuOptionPicked = DisplayMenu.ShowMenu<CrudMenuOptions>(crudMenu);
+            HandleMenuOptions.HandleCrudMenuOption(crudMenuOptionPicked, connectionString, ref continueRunning);
```

Comment:
Instead of passing a referenced parameter? Why didn't you just return a bool from the method and assign that to the local variable?
i.e.
```
continueRunning = HandleMenuOptions.HandleCrudMenuOptions(...);
```